src/components/Showcase.js

```diff
@@ -22,20 +22,20 @@ const StyledShowcase = styled("div")`
     z-index: 4;
   }
 
-  & > figure:nth-child(1) {
+  & > figure:nth-of-type(1) {
```

Comment:
Emotion was complaining about server-side rendering using nth-child and suggested changing to nth-of-type.